internal/core/input.rs

```diff
@@ -264,21 +264,6 @@ impl KeyEvent {
             };
         }
 
-        #[cfg(not(target_os = "macos"))]
-        {
-            if self.modifiers.control && !self.modifiers.alt && !self.modifiers.meta {
-                match keycode {
-                    key_codes::Home => {
-                        return Some(TextShortcut::Move(TextCursorDirection::StartOfText))
-                    }
-                    key_codes::End => {
-                        return Some(TextShortcut::Move(TextCursorDirection::EndOfText))
-                    }
-                    _ => (),
-                };
-            }
-        }
```

Comment:
Why do you remove this?

Reply:
That was not intentional. No idea what went wrong there. Sorry for that, the code is back now.